lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -906,7 +920,9 @@ public function getListTablesSQL()
     {
         // "sysdiagrams" table must be ignored as it's internal SQL Server table for Database Diagrams
         // Category 2 must be ignored as it is "MS SQL Server 'pseudo-system' object[s]" for replication
-        return "SELECT name FROM sysobjects WHERE type = 'U' AND name != 'sysdiagrams' AND category != 2 ORDER BY name";
+        return 'SELECT name FROM sys.sysobjects'
+            . ' WHERE type = \'U\' AND name != \'sysdiagrams\' AND category != 2'
+            . ' ORDER BY name';
```

Comment:
Ok so now we have backslashes and concatenation. Please use nowdoc instead.

Reply:
fine, fix it